includes/class-papercups.php

```diff
@@ -170,6 +170,7 @@ private function define_public_hooks() {
 
 		$plugin_public = new Papercups_Public( $this->get_plugin_name(), $this->get_version() );
 
+    $this->loader->add_action( 'wp_head', $plugin_public, 'header_script');
```

Comment:
@ZmagoD small nitpick, but it looks like the indentation is a bit off here -- is there a way we can auto-format everything in php (e.g. how `prettier` works for js/ts code?)

Reply:
Yes sure 👍 
I'm still waiting for the admin part to get values.

Reply:
sounds good makes sense!